### ISSUE TITLE
code is old

### ISSUE THREAD

**Issue by zhujun007007** [i-programmer/plyRPG#1]
hi, programmer,

now unity version is 5.6.1f1, plyGame version is 3.08, I think you should to update the code.

No matter how, thanks for your code,

**Comment by i-programmer** [comment]
Yep, I know about new versions of Unity and plyGame. But this code was for employers rather than anybody else)) So, I will not upgrade this code ever. 
Maybe I'll make new one someday with Unity2017 ¯\\\_(ツ)\_/¯
